Monika After Story/game/script-topics.rpy

```diff
@@ -12777,3 +12777,24 @@ label monika_sleigh:
     show monika 5hkbfa at t11 zorder MAS_MONIKA_Z with dissolve
     m 5hubfa "An experience like that with you would be so breathtaking~"
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_enjoyingspring",category=['spring'],prompt="Enjoying spring",random=mas_isSpring()))
+
+label monika_enjoyingspring:
+    m 1esa "Spring is such an amazing time of year, isn't it?"
+    m 1hua "The cold snow finally melts away, and the sunshine brings new life to nature."
+    m 1hub "When the flowers bloom, I can't help but smile!"
+    m 3hub "It's like the plants are waking up and saying 'hello world!' Ahaha~"
+    m 2esa "But I think the best thing about spring, would have to be the cherry blossoms."
+    m 4eub "They are very popular all around the world, but the most popular cherry blossoms would have to be the {i}'Somei Yoshino'{/i} in Japan."
+    m 3esb "Those ones in particular are naturally pure white, with a slight tinge of pink."
```

Comment:
I would say "mostly white" since if it's pure, it shouldn't have any other colors really.